server/dive_server/views_override.py

```diff
@@ -30,3 +32,30 @@ def use_private_queue(self, user: dict, privateQueueEnabled: bool):
             constants.UserPrivateQueueEnabledMarker, False
         ),
     }
+
+
+@access.user
+@boundHandler
+@autoDescribeRoute(Description('Get number of outstanding jobs'))
+def countJobs(self):
+    outstanding = (
+        job.Job()
+        .list(
+            user='all',
+            currentUser=self.getCurrentUser(),
+            statuses=[
+                GWJobStatus.QUEUED,
+                GWJobStatus.RUNNING,
+                GWJobStatus.CANCELING,
+                GWJobStatus.CONVERTING_INPUT,
+                GWJobStatus.CONVERTING_OUTPUT,
+                GWJobStatus.FETCHING_INPUT,
+                GWJobStatus.PUSHING_OUTPUT,
+            ],
+        )
+        .count()
+    )
+
+    return {
+        'outstanding': outstanding,
```

Comment:
"In front of" is kinda relative because there are 3 queues and the system doesn't really know how many queue processors are.  And then if you have 2 jobs (a transcode and a train) they'll be in different positions.

I don't think this would be very easy to explain to the user.

Reply:
Oh, and the reason for making it JSON is just that the rest of the API returns JSON and this seemed like a way to be able to add more information later without breaking stuff.  Not a lot of thought behind it.

Reply:
agreed, That would be difficult to indicate propery.